ProcessMaker/screen-builder
# FOUR-13078: Clicking in view all request shows server error

## Issue & Reproduction Steps

Expected behavior: 
When an user clicks in view all request it should see all the requests.
Actual behavior: 
 Clicking in view all request shows server error
## Solution
- improve empty form url
- add server url as a global parameter

## How to Test
Test the steps above

1. Create a new user.
2. log in with the user, and make sure you are in the default home page dashboard
3. If there are no requests or tasks available Click to view all requests or tasks
4. See the error.
5. If your user has tasks and/or requests you won’t see the links but you can also try these URLs:

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13078

## Code Review Checklist

ci:next
